src/rimeengine.cpp

```diff
@@ -224,6 +224,23 @@ RimeEngine::RimeEngine(Instance *instance)
     globalConfigReloadHandle_ = instance_->watchEvent(
         EventType::GlobalConfigReloaded, EventWatcherPhase::Default,
         [this](Event &) { refreshSessionPoolPolicy(); });
+    eventHandler_ = instance_->watchEvent(
```

Comment:
If you don't want it to be global even while you're using other input method, you should put it to ::keyEvent.